specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/modelManagement.json

```diff
@@ -0,0 +1,4755 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-05-01",
+    "title": "Azure Machine Learning Model Management Service",
+    "description": "These APIs allow end users to manage Azure Machine Learning Models, Images, Profiles, and Services."
+  },
+  "paths": {
+    "/modelmanagement/v{version}/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.MachineLearningServices/workspaces/{workspace}/assets": {
```

Comment:
What's the point of having a version as a parameter? The major version of the API will change causing changes in the schema returned by the services. The schema defined will likely not be applicable in the next major version. 

This should be a static part of the path, or at worst, a default value.

Reply:
Fix on the way for Modelmanagement service APIs to make the endpoints reflect similar structure as the other swaggers